single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/GraphMetadata.java

```diff
@@ -51,8 +53,21 @@ public static class NodeMetadata {
 
         private final List<NodeLabelMetadata> labels;
 
+        public NodeMetadata(@JsonProperty("id") String escapedId,
```

Comment:
True question: is there a need to keep this constructor? For example, for a `Node` that is not an `EquipmentNode`, the `equipmentId` is `null` and we do not have a specific method without the `equipmentId`.
Since the method is public, it would cause a breaking change to delete it.
Is deprecating it in the first place the right thing to do?